### DISCUSSION TITLE
How to backup and delete a blob once limits are crossed

### DISCUSSION THREAD

**Discussion by paramaggarwal** [vercel/community#5576] [Help] [answered]
### Page to Investigate

https://vercel.com/cbx1/~/stores/blob/store_jcHGJlRUNpardPZ1/settings

### Steps to Reproduce

How to delete a blob store once limit is reached?

It asks to delete the existing blobs, but there is no way to do so due to the limits getting crossed. What to do here?

<img width="1234" alt="Screenshot 2024-01-23 at 11 40 54 PM" src="https://github.com/vercel/community/assets/543981/7c576dd3-89e4-492b-8801-bb1c16133c65">

<img width="948" alt="Screenshot 2024-01-23 at 11 40 21 PM" src="https://github.com/vercel/community/assets/543981/fc3f3454-39f3-4cdb-a31a-056777f5fc3d">

**Comment by AviralJ58** [comment]
I am facing the same issue. Please let me know if you find a solution.

**Comment by swarnava** [comment, marked as answer]
Hi,

Once when you reach limits, we don't allow you to read or download files anymore. You can delete the file using Vercel SDK for the specific Blob storage.

**Reply by AviralJ58** [reply]
The UI to delete the blobs isn't accessible. I tried using the Node SDK to delete using the del() function but I can't access my blob store as it is suspended due to exceeded limits. How can I delete the blobs and delete the blob storage all together?

**Reply by swarnava** [reply]
Can you share the screenshot of the error message you are getting when trying to delete using SDK?

**Reply by AviralJ58** [reply]
![image](https://github.com/vercel/community/assets/77487252/0effbe38-4df0-4f83-893e-3a9df6df434e)
This is what I get when I try to list the blobs. I need to get the URLs of all the blobs to delete them using del(). Or is there an alternate way to delete all?

**Reply by paramaggarwal** [reply]
@swarnava It won't be possible for people to delete the files if they are unable to list the files. I think read+download should be blocked, but list+delete should be allowed - both from SDK and UI. Otherwise I feel that my data is being held hostage for payment. It hurts trust.

I understand that promoting conversion of free to paid plan is the expected resolution, but there should be a way to export my data and delete the bucket if I do not wish to move to a paid plan...
